Python 3 port

dotmpe opened the pull request:
I used Y-Digitals work [1] on porting the installer and made sure the application itself is running OK. Aside from the resource picker and graph rendering, this effectively ports the project to Python 3. [2]

1. https://github.com/Y-Digital/rdflib-web
2. https://github.com/dotmpe/rdflib-web

dotmpe committed:
Made install with pip3 work

- Does not fix any of the web-app for Python3.
- Fixed some blank lines/trailing whitespace as well.

Squashed commit of the following:

commit 64d0835b62f0601cabc4ed355eb9988bdd08ea38
Author: dr. Ian FitzPatrick <ian@ianfitzpatrick.eu>
Date:   Sat Jul 11 14:50:31 2020 +0200

    added re dependency

commit e1165452915a7f168196526dc408c3b7dd60a36c
Author: dr. Ian FitzPatrick <ian@ianfitzpatrick.eu>
Date:   Sat Jul 11 14:49:19 2020 +0200

    added back find_version

commit e14e30079e3779baee01fee4f13de76a24011836
Author: dr. Ian FitzPatrick <ian@ianfitzpatrick.eu>
Date:   Sat Jul 11 14:48:24 2020 +0200

    fix missing version

commit e8f101e7c4870983be363af947763ca72f79bf58
Author: dr. Ian FitzPatrick <ian@ianfitzpatrick.eu>
Date:   Sat Jul 11 14:47:19 2020 +0200

    added new minimal setup.py -- WIP

commit f2c5067fffea09f2aa22e0bc2ac404cf2a8c21f7
Author: dr. Ian FitzPatrick <ian@ianfitzpatrick.eu>
Date:   Sat Jul 11 14:36:31 2020 +0200

    automatic conversion using 2to3

dotmpe committed:
Made run with dev server

dotmpe committed:
Updated README>install a bit

dotmpe committed:
Other minor bits to run with Python3

dotmpe committed:
Final bit to make type-instance pages work

dotmpe committed:
Removed 'html' export option as plugin is not there

dotmpe committed:
Fixed encoding for bookdb

dotmpe committed:
Doc bits, todo's after python3 port

dotmpe committed:
Tasks

dotmpe committed:
Bumped version to 0.2 for python3 port

dotmpe committed:
Fixed download links

dotmpe committed:
Fixed graph renderer link

gromgull commented:
Thank you! 

there aren't many people interested in this project atm - if you want I can make you admin, and you can keep updating it now?

dotmpe commented:
I don't have any objections. 

I think this project is a nice stepping stone for people to start to play with RDF on the web and it is not as intimidating as Inrupt's React base.

dotmpe committed:
Before/after now run--again(?) (fixes EXECUTION-TIME)

Hooks where not called because blueprints are registered under third
blueprint.

Commit fixes task:
2021-03-24 FIXME: endpoint.{before,after}_request do not seem to be called with newer Flask?

gromgull commented:
Very good! I've made you admin! It's all yours - have fun!

dotmpe committed:
Fixed entry_points, HTML (query results) serializer

dotmpe committed:
Docs, tasks
